packs/sensu/README.md

```diff
@@ -1,10 +1,94 @@
-# Sensu Integration Pack
+#Sensu Integration Pack
 
-Pack which integrates with [sensu monitoring framework](http://sensuapp.org/).
+Integrates with beautiful [Sensu](http://sensuapp.org/) monitoring framework.
 
-## Sensors
+### Prerequisites
+A box with Sensu and StackStorm up and running. See installation for [Sensu](http://sensuapp.org/docs/latest/guide) and [StackStorm](http://docs.stackstorm.com/install).
```

Comment:
instead of StackStorm you can use `|st2|` and then magic happens. Should apply everywhere.